shared/src/constants/constants.ts

```diff
@@ -19,19 +19,41 @@ export enum CHAINID {
   AVAX_MAINNET = 43114,
 }
 
-export type NETWORK_NAMES  = "mainnet" | "kovan" | "fuji" | "avax";
+export type NETWORK_NAMES = "mainnet" | "kovan" | "fuji" | "avax";
 export const NETWORKS: Record<number, NETWORK_NAMES> = {
   [CHAINID.ETH_MAINNET]: "mainnet",
   [CHAINID.ETH_KOVAN]: "kovan",
   [CHAINID.AVAX_FUJI]: "fuji",
   [CHAINID.AVAX_MAINNET]: "avax",
 };
 
+// TODO: Remove the isProduction check when enabling avalanche
+export const ENABLED_CHAINID: CHAINID[] = isProduction()
+  ? [CHAINID.ETH_MAINNET]
+  : [CHAINID.ETH_MAINNET, CHAINID.AVAX_MAINNET];
```

Comment:
nit: Could add `isDevelopment()` so it works in localhost but not a big deal.

Reply:
This is how we normally do feature flagging